doc/user/content/manage/user-management/rbac-alterroles.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Update and drop roles"
+description: ""
+menu:
+  main:
+    parent: user-management
+    weight: 16
+---
+
+In the [previous guide](/manage/user-management/rbac-newrole), you created a new role and assigned privileges for
+specific objects. This guide walks you through updating your previously created
+roles and removing users or roles when they are no longer needed.
+
+## Before you begin
+
+Make sure you have already created a new user and a role.
+
+## Step 1. Create a second role
+
+For this tutorial, you will create a new role with different privileges to other objects.
+Then you will apply those privileges to the `dev` role and alter or drop
+privileges as needed.
+
+1. Create a second role your Materialize account:
+
+   ```sql
+   CREATE ROLE qa WITH CREATEDB;
+   ```
+
+   This role has permission to create a new database in the Materialize account.
+
+2. Create a new `qa_db` database:
+
+   ```sql
+   CREATE DATABASE qa_db;
+   ```
+
+3. Apply `USAGE` and `CREATE` privileges to the `qa_role` role for the new database:
+
+   ```sql
+   GRANT USAGE, CREATE ON DATABASE qa_db TO qa_role;
+   ```
+
+## Step 2. Add inherited privileges to a role
+
+Your `dev_role` also needs access to `qa_db`. You can apply these
+privileges individually or you can choose to grant the `dev_role` the same
+permissions as the `qa_role`.
+
+1. Add `dev_role` as a member of `qa_role`:
+
+   ```sql
+   GRANT qa_role TO dev_role;
+   ```
+
+   Roles also inherit all the attributes and privileges of the granted role.
+   Making roles members of other roles allows you to manage sets of
+   permissions, rather than granting privileges to roles on an individual basis.
+
+2. Review the privileges of `qa_role` and `dev_role`:
+
+   ```sql
+   SELECT name, privileges FROM mz_databases WHERE name='qa_db';
+   ```
+
+   Your output will be similar to the example below:
+
+   ```nofmt
+   name|privileges
+   qa_db|{u1=UC/u1,u9=UC/u1}
+   (1 row)
+   ```
+
+   Both `dev_role` and `qa_role` have usage and create access to the `qa_db`. In
+   the next section, you will edit role attributes for these roles and drop
+   privileges.
+
+## Step 3. Revoke privileges and alter role attributes
+
+Your `dev_role` and `qa_role` have the same role attributes. You can alter or
+revoke certain attributes and privileges for each role, even if they are
+inherited from another role.
+
+1. Update the `CREATEROLE` attribute for the `dev_role`:
+
+   ```sql
+   ALTER ROLE dev WITH CREATEROLE;
+   ```
+
+   The `qa_role` will not have the `CREATEROLE` attribute enabled because the
+   attribute was enabled _after_ the role grant.
```

Comment:
I don't think this is accurate. Attributes are not inherited no matter what order they are granted in.